pkg/controllers/provisioningv2/rke2/bootstrap/controller.go

```diff
@@ -109,7 +109,7 @@ func (h *handler) getBootstrapSecret(namespace, name string, envVars []corev1.En
 		}
 
 		hash := sha256.Sum256(secret.Data["token"])
-		data, err := installer.LinuxInstallScript(base64.URLEncoding.EncodeToString(hash[:]), envVars, "")
+		data, err := installer.LinuxInstallScript(context.Background(), base64.URLEncoding.EncodeToString(hash[:]), envVars, "")
```

Comment:
Does this not mean that when we are provisioning machines via machine, these machines will not get the proper CA checksum populated into the install script?

Reply:
Only if hostPort is set. This is currently the case as well (in other words, this is not a regression). I will see if I can fix this here too.